src/CMakeLists.txt

```diff
@@ -196,9 +203,14 @@ if (CURSES)
     endif ()
 
     target_include_directories(cataclysm-common PUBLIC
-        ${CURSES_INCLUDE_DIR} ${NCURSES_INCLUDE_DIRS} ${ZLIB_INCLUDE_DIRS}
+        ${CURSES_INCLUDE_DIR} ${NCURSES_INCLUDE_DIRS}
     )
-    target_link_libraries(cataclysm-common PUBLIC ${CURSES_LIBRARIES} ${ZLIB_LIBRARIES})
+    target_link_libraries(cataclysm-common PUBLIC ${CURSES_LIBRARIES})
+    if (TARGET ZLIB::ZLIB)
+        target_link_libraries(cataclysm-common PUBLIC ZLIB::ZLIB)
+    else()
+        target_link_libraries(cataclysm-common PUBLIC ${ZLIB_LIBRARIES})
+    endif()
```

Comment:
This seems identical to the ZLIB linkage block above, and not doing anything extra. Debug leftover?

Reply:
No, but good point, I will move it outside.